src/interface/src/app/plan/project-areas-metrics/project-areas-metrics.component.html

```diff
@@ -4,17 +4,23 @@ <h3 class="title">Metrics per Project area</h3>
 </div>
 <div class="chart-holder">
   <div *ngFor="let chart of selectedCharts; index as i">
-    <mat-form-field>
-      <mat-select
-        [value]="chart"
-        (selectionChange)="selectDataPoint($event, i)">
-        <mat-option
-          *ngFor="let dataSet of filterData(data, selectedCharts, chart)"
-          [value]="dataSet">
-          {{ dataSet.label }}
-        </mat-option>
-      </mat-select>
-    </mat-form-field>
+    <!-- TODO Add styling class -->
```

Comment:
This todo is an easy one to do on this pr :)

Reply:
You caught me being lazy 😅 . Added a couple styling classes